app/routes/circulars._archive._index/route.tsx

```diff
@@ -75,14 +81,54 @@ export default function () {
   if (searchString) searchString = `?${searchString}`
 
   const [inputQuery, setInputQuery] = useState(query)
-  const clean = inputQuery === query
+  const [inputDateGte, setInputDateGte] = useState(startDate)
+  const [inputDateLte, setInputDateLte] = useState(endDate)
+  const [showContent, setShowContent] = useState(false)
+  const [showDateRange, setShowDateRange] = useState(false)
+  const clean =
+    inputQuery === query &&
+    inputDateGte === startDate &&
+    inputDateLte === endDate
```

Comment:
I suggest that we don't want to wipe away the search results if the date range is dirty (unsubmitted).

```suggestion
  const clean = inputQuery === query
```